public/components/common/__test__/__snapshots__/header.test.tsx.snap

```diff
@@ -70,47 +70,10 @@ exports[`Header component Renders header component 1`] = `
                 </EuiSpacer>
                 <EuiCallOut
                   iconType="iInCircle"
-                  title="Experimental Feature"
                 >
                   <div
                     className="euiCallOut euiCallOut--primary"
                   >
-                    <div
-                      className="euiCallOutHeader"
-                    >
```

Comment:
I think EuiCallOut compoent is specifically used for Experimental Feature. If we want to keep the text, then just leave it as a EuiText